create/how_to/aws.md

```diff
@@ -15,23 +15,23 @@ The following guide will walk you through every step to deploy MeiliSearch in an
 
 After logging into your [AWS Console](https://aws.amazon.com/console), navigate to the **Compute** service. Then go to **EC2**, and finally open your **Instances** console.
 
-![Launch an instance from AWS console](/aws/01.launch-instances.png)
+![Launching an instance from AWS console](/aws/01.launch-instances.png)
```

Comment:
Rather than just removing alt text entirely, you could also focus alt text on the title of each AWS page (useful for confirming they're on the right page) and any values that have been selected on that page that are not mentioned explicitly in the content of that section.

Reply:
>Rather than just removing alt text entirely, you could also focus alt text on the title of each AWS page (useful for confirming they're on the right page) and any values that have been selected on that page that are not mentioned explicitly in the content of that step.

Could you give an example please?

Reply:
Sure, here's an example for step 1 of AWS. "Page titled instances. Text in center of screen: You do not have any instances in this region."

You have to make a judgment call about which information is valuable. In this case, since the "Launch instances" button is mentioned explicitly in the text below, I did not call attention to it. I also deemed the sidebar, navbar, etc. to be not useful.